packages/generators/init-generator.ts

```diff
@@ -40,16 +43,63 @@ export default class InitGenerator extends Generator {
 		super(args, opts);
 		this.isProd = false;
 		this.usingDefaults = false,
+
 		this.dependencies = [
 			"webpack",
 			"webpack-cli",
-			"terser-webpack-plugin",
+			"webpack-dev-server",
 			"babel-plugin-syntax-dynamic-import",
 		];
+		if (this.isProd) {
+			this.dependencies.push("terser-webpack-plugin");
+		}
+
 		this.configuration = {
 			config: {
 				topScope: [],
-				webpackOptions: {},
+				// tslint:disable: object-literal-sort-keys
```

Comment:
don't disable rules

Reply:
I'm trying to make the generated webpack configuration "standard", where for standard I mean with keys ordered like [this](https://webpack.js.org/configuration#options):

```js
module.exports = {
  mode: "production", // "production" | "development" | "none"
  entry: "./app/entry", // string | object | array
  output: {
    /* ... */
  },
  module: {
    /* ... */
  }
```

The other solution I see is to keep the keys ordered is to set them this way: 

```js
webpackOptions.mode = ...
webpackOptions.entry = ...
webpackOption.output = ...
```

Reply:
It makes sense to me, that's the order I've been using for ages

Reply:
I'm keeping this if it's ok